numismatic/feeds/poloniex.py

```diff
@@ -35,7 +35,7 @@ class PoloniexWebsocketClient(WebsocketClient):
 
         packet = json.dumps(connection_message)
         logger.info(packet)
-        await self.websocket.send(packet)
+        await PoloniexWebsocketClient.websocket.send(packet)
```

Comment:
Wuth `@property` on the base, we can get away using `self.websocket` while still adhering to the singleton. I suppose if you like (and agree) with my method, then I will implement that and everything should work